com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/AndroidBundler.java

```diff
@@ -137,8 +137,6 @@ private static void initAndroid() {
                 Bob.atomicCopy(libc_url, f, false);
             }
 
-            Bob.extract(Bob.class.getResource("/lib/android-res.zip"), rootFolder);
```

Comment:
this archive is empty for a few years, because all the resources were removed from the engine. we don't need it anymore